docs/docs_for_team/README.md

```diff
@@ -73,6 +73,10 @@ From Jenkins, 'New item' -> 'Pipeline'. Then, the settings are as follows:
 * SCM: Git
 * Pipeline Repository URL: [git-repo-url]
 
+## Known bugs
+
+As it stands configuration is loaded from your local environment using a [cloud init yaml file](terraform/jenkins/cloud-init/server-asg-xenial-16.04-amd64-server.yaml). In order to make sure that your latest changes are sent to terraform you must update the [Jenkins variables file](terraform/jenkins/variables.tf) with the name of your working branch.
```

Comment:
The links don't work here but I'm 99% sure that they will once this is merged

Reply:
@JonathanHallam Thanks for this. Based on our chat on Friday, shall we reword this by mentioning that it applies to changes to the `docker` folder and related files?